src/pybind/mgr/cephadm/inventory.py

```diff
@@ -958,7 +958,25 @@ def get_schedulable_hosts(self) -> List[HostSpec]:
             h for h in self.mgr.inventory.all_specs()
             if (
                 self.host_had_daemon_refresh(h.hostname)
-                and '_no_schedule' not in h.labels
+                and SpecialHostLabels.drain_daemons_label not in h.labels
+            )
+        ]
+
+    def get_conf_keyring_available_hosts(self) -> List[HostSpec]:
+        """
+        Returns all hosts without _no_conf_keyring label that
+        have had a refresh
```

Comment:
I think the comment should be explicit about what hosts is getting. Something like:

`Returns all hosts eligible to deploy conf and keyring files (without _no_conf_keyring label)`

Reply:
added a sentence like that to the comment